### PR TITLE
WIP: Small tweaks

### PR DESCRIPTION
**Pull request by thijs-obj**

### CODE REVIEW

File: analysis/README.md [hunk 1]
```diff
@@ -9,13 +9,7 @@ export PYTHONPATH=.
...
 # This will fail if the postgres lib development headers are not present if so, then on Ubuntu that can
 # be fixed with: sudo apt-get install libpq-dev
-
-# install bach and bach_open_taxonomy in edit mode, so local changes are immediately useable
-pip install -e ../bach
-pip install -e bach_open_taxonomy
-
-# needed for the product_analytics.ipynb notebook
-pip install jupyter plotly matplotlib jupyter-dash seaborn 
+pip install -r requirements.txt
```
Comment (thijs-obj) [inline analysis/README.md:12]: Was getting everything up and running earlier today, and I figured this was a bit easier.

File: analysis/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# install bach and bach_open_taxonomy in edit mode, so local changes are immediately usable
+-e ../bach
+-e bach_open_taxonomy
+
+# requirements for the product_analytics.ipynb notebook
+jupyter
+plotly
+matplotlib
+jupyter-dash
+seaborn
```
Comment (jansentom) [inline analysis/requirements.txt:10]:
```suggestion
jupyter
jupyter-dash
matplotlib
plotly
seaborn
```

**Review by jansentom — APPROVED** [review]

**Comment by jansentom** [comment]
Can we merge this @thijs-obj ?

**Comment by thijs-obj** [comment]
I think so

**Pull request merged**